Gruntfile.js

```diff
@@ -0,0 +1,169 @@
+          paths: getLoadPaths,
+          compress: true,
+          modifyVars: {
+            'fa-font-path': '"fonts"',
+            'img-path': '"../images"'
+          }
+        },
+        files: [{
+          expand: true,
+          src: "themes/*/less/compiled.less",
+          rename: function (dest, src) {
+            return src.replace('/less/', '/css/').replace('.less', '.css');
+          }
+        }]
+      }
+    },
+    // SASS compilation
+    scss: {
+      sass: {
+        options: {
+          style: 'compress'
+        }
+      }
+    },
+    // Convert LESS to SASS, mostly for development team use
+    lessToSass: {
+      convert: {
+        files: [
+          {
+            expand: true,
+            cwd: 'themes/bootstrap3/less',
+            src: ['*.less', 'components/*.less'],
+            ext: '.scss',
+            dest: 'themes/bootstrap3/scss'
+          },
+          {
+            expand: true,
+            cwd: 'themes/bootprint3/less',
+            src: ['*.less'],
+            ext: '.scss',
+            dest: 'themes/bootprint3/scss'
+          }
+        ],
+        options: {
+          replacements: [
+            { // Replace ; in include with ,
+              pattern: /(\s+)@include ([^\(]+)\(([^\)]+)\);/gi,
+              replacement: function mixinCommas(match, space, $1, $2) {
+                return space + '@include ' + $1 + '(' + $2.replace(/;/g, ',') + ');';
+              },
+              order: 3
+            },
+            { // Inline &:extends converted
+              pattern: /&:extend\(([^\)]+)\)/gi,
+              replacement: '@extend $1',
+              order: 3
+            },
+            { // Inline variables not default
+              pattern: / !default; }/gi,
+              replacement: '; }',
+              order: 3
+            },
+            {  // VuFind: Correct paths
+              pattern: 'vendor/bootstrap/bootstrap',
+              replacement: 'vendor/bootstrap',
+              order: 4
+            },
+            {
+              pattern: '$fa-font-path: "../../../fonts" !default;',
+              replacement: '$fa-font-path: "fonts";',
+              order: 4
+            },
+            {
+              pattern: '$img-path: "../../images" !default;',
+              replacement: '$img-path: "../images";',
+              order: 4
+            },
+            { // VuFind: Bootprint fixes
+              pattern: '@import "bootstrap";\n@import "variables";',
+              replacement: '@import "variables", "bootstrap";',
+              order: 4
+            },
+            {
+              pattern: '$brand-primary: #619144 !default;',
+              replacement: '$brand-primary: #619144;',
+              order: 4
+            }
+          ]
+        }
+      }
+    },
+    watch: {
+      options: {
+        atBegin: true
+      },
+      less: {
+        files: 'themes/**/*.less',
+        tasks: ['less']
+      },
+      scss: {
+        files: 'themes/**/*.scss',
```

Comment:
Hi Chris, we just ran into an issue that grunt-watch was throwing "Maximum call stack size exceeded" warnings resulting in not compiling the scss files it's supposed to. We limited the path further down with the pattern `'themes/*/scss/**/*.scss'` as all scss files should reside in the themes scss folder and instantly got rid of the warnings. This pattern enforces the folder structure and reduces the files grunt-watch is watching - maybe something to consider for this pull request as well and the pattern could be applied to less, too.